features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,16 @@
-# put step definitions in this file
\ No newline at end of file
+Given(/^I visit the index page$/) do
+  visit '/'
+end
+
+Then(/^I would like to see the text "([^"]*)"$/) do |text|
+  expect(page).to have_content text
+end
+
+Given(/^I visit the Amber page$/) do
+  visit '/name/Amber'
+end
```

Comment:
This step is not quite right. I would prefer if you pass in a name in this step definition and use something like this:
```
visit "/name/#{name}"
```
Does that make sense?